codice/alliance
# CAL-65 Added Sources to Query tab to the NSILI App on the Admin UI so that a system administrator can configure the sources to perform a federated query

#### What does this PR do?
This PR Added Sources to Query tab to the NSILI App on the Admin UI so that a system administrator can configure the sources to perform a federated query.
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@glenhein @peterhuffer @bdeining @troymohl 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@beyelerb 
@kcwire
#### How should this be tested?

1. Run the JUnit tests.
2. Confirm that 
 - adding and removing query sources from the `NSILI Endpoint Sources to Query` list on the Sources to Query tab on the NSILI App work as expected (see https://github.com/codice/alliance/pull/65).
 - the Queryable Sources and Queried Sources are correctly dynamically populated.

#### Any background context you want to provide?
This PR moves the functionality from https://github.com/codice/alliance/pull/65 to a new tab so that the sources to query are dynamically populated instead of manually entered.
#### What are the relevant tickets?
https://codice.atlassian.net/browse/CAL-65
#### Screenshots (if appropriate):
![image](https://cloud.githubusercontent.com/assets/8041246/18364446/2c3e464c-75c3-11e6-82e3-379bbc1caed7.png)
![image](https://cloud.githubusercontent.com/assets/8041246/18364449/30a2ff7a-75c3-11e6-938b-b05f3f2795cd.png)
![image](https://cloud.githubusercontent.com/assets/8041246/18364454/34007e72-75c3-11e6-9a4f-d37e289bad30.png)
![image](https://cloud.githubusercontent.com/assets/8041246/18364463/38698972-75c3-11e6-938b-210ad881aeef.png)
![image](https://cloud.githubusercontent.com/assets/8041246/18444872/e1375674-78cf-11e6-9048-264f4cef0ea1.png)
#### Checklist:
